charts/prometheus-cloudwatch-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "0.10.0"
 description: A Helm chart for prometheus cloudwatch-exporter
 name: prometheus-cloudwatch-exporter
-version: 0.14.1
+version: 0.15.0
```

Comment:
@joshuastern I just merged another PR could you bump the chart version. Your change looks good!

Reply:
bumped!